### DISCUSSION TITLE
Emmet support for HTML files

### DISCUSSION THREAD

**Discussion by jainam03** [SpartanJ/ecode#16] [Q&A] [answered]
Does ecode code editor have emmet abbreviation support for HTML files????

**Comment by SpartanJ** [comment, marked as answer]
Hi @jainam03! Not for the moment. It would be cool but there are many things in the backlog first. Of course, this could be provided as a plugin, I hope some collaborators appear soon so we can improve the speed of the development.

**Reply by jainam03** [reply]
Ohh yeah right... I understand...good work anyways!!👏🏻
